py/sentry_streams/user_functions/agg_template.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+from typing import Any
+
+
+class Accumulator(ABC):
```

Comment:
Do you have a sense on which accumulators we will provide out of the box ?

Reply:
Not yet. I think that might be a good idea for next PR. Leaving a TODO